chroma_core/south_migrations/0048_auto__add_field_stratagemconfiguration_not_deleted.py

```diff
@@ -0,0 +1,1515 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding field 'StratagemConfiguration.not_deleted'
+        db.add_column(u'chroma_core_stratagemconfiguration', 'not_deleted',
+                      self.gf('django.db.models.fields.NullBooleanField')(default=True, null=True, blank=True),
```

Comment:
could this be in 0047?

Reply:
Should be in 0046 if anything. I'll take a look

Reply:
ok awesome.